risk_module/settings.py

```diff
@@ -11,27 +11,51 @@
 """
 
 import os
+import environ
 from pathlib import Path
 from celery.schedules import crontab
 
-# Build paths inside the project like this: BASE_DIR / 'subdir'.
-BASE_DIR = Path(__file__).resolve().parent.parent
+from risk_module import sentry
 
 
-# Quick-start development settings - unsuitable for production
-# See https://docs.djangoproject.com/en/3.2/howto/deployment/checklist/
+env = environ.Env(
+    DJANGO_DEBUG=(bool, False),
+    DJANGO_SECRET_KEY=str,
+    DJANGO_ALLOWED_HOST=(str, '*'),
+    # Database
+    DATABASE_NAME=str,
+    DATABASE_USER=str,
+    DATABASE_PASSWORD=str,
+    DATABASE_PORT=str,
+    DATABASE_HOST=str,
+    TIME_ZONE=(str, 'UTC'),
+
+    USE_AWS_FOR_MEDIA=(bool, False),
+    S3_AWS_ACCESS_KEY_ID=str,
+    S3_AWS_SECRET_ACCESS_KEY=str,
+    S3_STORAGE_BUCKET_NAME=str,
+    S3_REGION_NAME=str,
+
+    CELERY_REDIS_URL=(str, 'redis://redis:6379/0'),
```

Comment:
Let's define this in docker-compose.yml instead
```suggestion
    CELERY_REDIS_URL=str,
```